### DISCUSSION TITLE
Chart grid lines

### DISCUSSION THREAD

**Discussion by Eagleleader** [vikramlearning/blazorbootstrap#823] [Q&A] [answered]
Hello, is there a way to remove/hide the lines of the chart ?
In the documentation of Chart.JS there is 2 ways to do it : setting the color of the lines (which I would prefer to avoid because of maintenance issues) and simply apply a boolean to show them or not :

```
var options = {
    scales: {
        xAxes: [{
            gridLines: {
                display:false
            }
        }],
        yAxes: [{
            gridLines: {
                display:false
            }   
        }]
    }
}
```

I don't find any possibility into the BarChart to hide X or Y lines.
Is it possible to do it ?

Thanks in advance for your time and answers

**Comment by gvreddy04** [comment, marked as answer]
@Eagleleader Thank you for using **BlazorBootstrap**. We have already included this feature in the recent preview versions. I think the demo example is missing. I will include it in the next version.

Link: https://github.com/vikramlearning/blazorbootstrap/blob/cbf1b523ca29c29f3467da9f77c3e7bca892f1bb/blazorbootstrap/Models/Charts/ChartOptions/ChartOptions.cs#L317

I'll try to provide an example here.

**Reply by Eagleleader** [reply]
Hi @gvreddy04, thank you for this answer. Also I had to force Visual Studio to include preview versions in order to have it. Otherwise I was still using version 2.2.0
I had to make some small changes into the code to use new enums or List<double?>insted of List<double> but everything is working well. 
I like your project which I use in complement of MudBlazor because I prefer the ChartJs charts.
I hope this project will go as far as it can !
Good work and continue like this !
